openshift/template.yaml

```diff
@@ -318,3 +318,14 @@ objects:
         targetPort: 8090
     selector:
       app: assisted-service
+- apiVersion: policy/v1
```

Comment:
I wonder if that's the right apiVersion for the cluster version that we use, because in this example they used betav1:
https://github.com/app-sre/github-mirror/blob/e03ea308941d1dbc14f9530a4c7b751b3e45936b/openshift/github-mirror.yaml#L7